osu.Game.Modes.Taiko/Objects/Drawable/DrawableDrumRollTick.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Input;
+using osu.Game.Modes.Taiko.Judgements;
+using System;
+using osu.Game.Modes.Objects.Drawables;
+
+namespace osu.Game.Modes.Taiko.Objects.Drawable
+{
+    public class DrawableDrumRollTick : DrawableTaikoHitObject
+    {
+        private readonly DrumRollTick tick;
+
+        public DrawableDrumRollTick(DrumRollTick tick)
+            : base(tick)
+        {
+            this.tick = tick;
+        }
+
+        protected override TaikoJudgement CreateJudgement() => new TaikoDrumRollTickJudgement();
+
+        protected override void CheckJudgement(bool userTriggered)
+        {
+            if (!userTriggered)
+            {
+                if (Judgement.TimeOffset > tick.TickTimeDistance / 2)

```

Comment:
Re: Abs check: No, otherwise that may cause them to be missed before they touch the hit marker.

Re: TickHitWindow: Fair idea.